go/pkg/chunker/chunker.go

```diff
@@ -23,13 +24,13 @@ var ErrEOF = errors.New("ErrEOF")
 // Chunker can be used to chunk an input into uploadable-size byte slices.
 // A single Chunker is NOT thread-safe; it should be used by a single uploader thread.
 type Chunker struct {
-	chunkSize        int
-	reader           io.ReadCloser
-	contents, buffer []byte
-	digest           digest.Digest
-	offset           int64
-	initialized      bool
-	path             string
+	chunkSize   int
+	reader      *bufio.Reader
+	contents    []byte
```

Comment:
nit: comment that contents caches small files that are read from buffered io in one chunk?